src/software/gui/drawing/ball.cpp

```diff
@@ -11,28 +15,43 @@ void drawBallVelocity(QGraphicsScene *scene, const Ball &ball, const QColor &col
     pen.setCapStyle(Qt::PenCapStyle::RoundCap);
     pen.setCosmetic(true);
 
-    drawSegment(scene, Segment(ball.position(), ball.position() + ball.velocity()), pen);
+    drawSegment(scene, Segment(position, position + velocity), pen);
 }
 
-void drawBallPosition(QGraphicsScene *scene, const Ball &ball, const QColor &color)
+void drawBallPosition(QGraphicsScene *scene, const Point &position, const double height,
+                      const QColor &color)
 {
     QPen pen(color);
     pen.setWidth(2);
     pen.setCosmetic(true);
 
+    QColor fill_colour = color;
+    // Decrease the alpha value as the ball's height increases
```

Comment:
why do we make the ball more transparent as it goes up? is it just to create an effect?

Reply:
Yea, I haven't tested completely yet but it's to give a visual representation of when the ball is in the air.